### ISSUE TITLE
Report only test result in APA format?

### ISSUE THREAD

**Issue by LukasWallrich** [easystats/report#86]
**Describe the solution you'd like**
This package is very helpful - thanks a lot! However, it would be great if it could also just return the results of a test properly formatted, without any explanatory text - typically, that is presently returned in brackets somewhere in the middle of the long text, for instance for a t-test that would be t(86.54) = -10.52, p < .001

**How could we do it?**
Maybe as an additional item of the return list? I know that this functionality is already implemented in some other packages such as `apa`, but it would be great to be able to use `report` for all reporting ...

**Comment by DominiqueMakowski** [comment]
That's a good idea. In fact, if you have followed the development of report a bit, you'll see that the API has changed quite a bit, I went through backs and forths and to be honest I'm still not 100% of what is the best route to build these reports.

Essentially, I would like it to be as modular and "assemblage-based" as possible, even though sometimes that complicates the process more than simplify it.

As you might know, report relies mainly on 3 easystats packages, parameters, performance and effectsize, to compute all the useful components. Then, I think that the first step should be to assemble all the indices into tables, and from there an intermediate step could be to have a function creating the "parameters reports" (could have a function called `report_parameters` or something) that would do have you have described above. These would be then inserted into text strings for the full report.

Anyway, this package could definitely be improved a lot, and benefit from a fresh eye on it. Right now my time is limited but I hope to be able to work this out someday, but if you're looking for a project to get involved in, by all means let us know :)

**Comment by strengejacke** [comment]
@DominiqueMakowski Any idea how we should name this function / element? We have `text_*()` for the textual output, or `report_*()` for specific parts. Could imagine something like `report_apa()` or `report_statistic(style = "apa")` or something.

**Comment by DominiqueMakowski** [comment]
yes `report_statistics()`? makes sense I think, indeed with a `style="apa"` arg

**Comment by DominiqueMakowski** [comment]
or `text_statistics`... 

The problem is that currently we have `text_*`, `table_*` and `report_*` is most of the times an integration of both and allows to produce both forms. So maybe yes `text_statistics` is appropriate? and `text_parameters()` would be the full sentence including interpretations for the parameters?

**Comment by strengejacke** [comment]
It depends on where we store the information. Looking at this issue, and at #28, the general idea is:

* `report()` create a generic, comprehensive report
* `report_*()` creates more specific reports
* `text_()* creates text output from the generic report

That's quite compact and intuitive. Thus, in my understanding, `text_*()` should never do something directly with objects like regression models, but work on report-objects. So I would say, `report_statistics()` makes sense, like we have`report_parameters()` etc.

**Comment by bwiernik** [comment]
(Strictly speaking, APA style calls for an effect size and confidence interval to always be reported with the test statistic.)

**Comment by LukasWallrich** [comment]
> (Strictly speaking, APA style calls for an effect size and confidence interval to always be reported with the test statistic.)

That is certainly true - but just reporting this all at once can lead to a rather unwieldy text - thus my initial suggestion to offer a shorter output for the fully codified bit.
